tests/Integration/Schema/Directives/WithSumDirectiveTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Tests\Integration\Schema\Directives;
+
+use Illuminate\Support\Facades\DB;
+use Nuwave\Lighthouse\Exceptions\DefinitionException;
+use Tests\DBTestCase;
+use Tests\Utils\Models\Comment;
+use Tests\Utils\Models\Post;
+
+class WithSumDirectiveTest extends DBTestCase
+{
+    public function testEagerLoadsRelationSum(): void
+    {
+        $this->schema = /** @lang GraphQL */ '
+        type Query {
+            posts: [Post!] @all
+        }
+
+        type Post {
+            comments_sum_votes: Int!
```

Comment:
This use case is pretty much precisely what the `@with*` directives are explicitly *not* meant for, see the description of `@withCount`. There should be a `@sum` directive for this.